monitoring/user-values-prom-operator.yaml

```diff
@@ -64,6 +64,8 @@
 #   enabled: true
 #   adminPassword: yourSecretPasswordHere
 #   "grafana.ini":
+#     initChownData:
```

Comment:
Is initChownData part of the "grafana.ini" file?  The indentation here makes that look like the case.  However, if I look at the Grafana Helm chart, it looks like the initChownData stanza is a top-level item (i.e. w/no indentation)...which would mean it should be indented one-level below the grafana: tag here in this chart.

Reply:
I moved the parameter and it is working as expected now.